client/jetpack-cloud/sections/agency-dashboard/sites-overview/index.tsx

```diff
@@ -124,6 +133,43 @@ export default function SitesOverview() {
 
 	const isFavoritesTab = selectedTab.key === 'favorites';
 
+	const issueLicenseRedirectUrl = useMemo( () => {
+		// Convert the product type (e.g., backup, scan) to comma-separated product slugs
+		// that can be used to issue multiple licenses.
+		const commaSeparatedProductSlugs = selectedLicenses
+			?.map( ( type: string ) => getProductSlugFromProductType( type ) )
+			.join( ',' );
+
+		return `/partner-portal/issue-license/?site_id=${ selectedLicensesSiteId }&product_slug=${ commaSeparatedProductSlugs }&source=dashboard`;
```

Comment:
We can use `addQueryArgs` to construct this URL. The signature is `addQueryArgs( query, path );` where `query` is an object of the query parameters we want to add to `path`.

Reply:
Thanks for suggesting this! I have also used the `addQueryArgs` to handle the array itself (without forcing comma-separated values).